start.sh

```diff
@@ -55,7 +55,7 @@ cp /blob-stream-inclusion/blob_inclusion/script/proof-with-pis.json /output/blob
 
 echo "Running e2e_blobstream_inclusion.sh"
 echo
-export MAX_THREADS=4
+export MAX_THREADS=${MAX_THREADS:-4}
```

Comment:
`./e2e_blobstream_inclusion.sh`

This script in the other repo overrides the max threads env, so this change is good, but also requires a change in that repo to not override the env if it's already set.

Reply:
Thanks!